content/en/real_user_monitoring/browser/monitoring_page_performance.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Monitoring Page Performance
+kind: documentation
+further_reading:
+  - link: "https://www.datadoghq.com/blog/real-user-monitoring-with-datadog/"
+    tag: "Blog"
+    text: "Real User Monitoring"
+  - link: "/real_user_monitoring/explorer/"
+    tag: "Documentation"
+    text: "Explore your views within Datadog"
+  - link: "/real_user_monitoring/explorer/analytics/"
+    tag: "Documentation"
+    text: "Build analytics upon your events"
+  - link: "/real_user_monitoring/dashboards/"
+    tag: "Documentation"
+    text: "RUM Dashboards"
+---
+
+When a user visits a page on your website, a new **RUM view event** is created. The view contains live page load and performance metrics. Performance metrics are sent to Datadog as the page finishes loading, but might be empty for a few seconds before they appear in the Datadog UI.
+
+## Identifying performance bottlenecks
+
+To find what's causing a page to under-perform:
+
+1. Set up and configure [RUM Browser monitoring][1] for your application.
+2. Open the [Performance Overview dashboard][2]. It provides you with a high level view of your monitored pages, and you can apply filters to find pages with performance problems.
+3. When you find a page with problem, click the event to open it in the [RUM Explorer][3].
+4. Open the event side panel to see what is causing the performance issue with the help of the waterfall.
+
+## Performance metrics for views
+
+Performance metrics are collected for each individual view. These metrics can pinpoint exactly where the problem is occurring when used with the RUM context collected by default (current page view information, geoIP data, browser information, etc.) and extended with [Global Context][4].
+
+These performance metrics are key for helping you start your investigations:
+
+- **Time To First Byte:** How long it takes the server to process the request. If the server is slow, try using APM to understand what's causing the slowness on the server side.
+- **First Contentful Paint:** How long until something is displayed. Check the RUM waterfall for blocking resources and long tasks preventing the browser from rendering content.
+- **Loading Time:** How long until the page is interactive. Check the RUM waterfall to see if you're loading too many assets, or if some resources are blocking the rendering.
+
+## All performance metrics
+
+For information about the default attributes for all RUM event types, see [Data Collected][5]. For information about configuring for sampling, global context, or custom user actions and custom errors, see [Advanced Configuration][6]. The following table lists the performance metrics collected from [Navigation Timing API][7] and [Paint Timing API][8]:
```

Comment:
"The following table lists **Datadog specific metrics along with performance metrics** collected from [Navigation Timing API][7] and [Paint Timing API][8]:" or similar. We also come up with some in-house metrics to help the filtering (error count, action count,...)